doc/src/manual/control-flow.md

```diff
@@ -409,7 +409,7 @@ julia> for i = 1:5
 5
 ```
 
-Here the `1:5` is a `Range` object, representing the sequence of numbers 1, 2, 3, 4, 5. The `for`
+Here the `1:5` is an `AbstractRange` object, representing the sequence of numbers 1, 2, 3, 4, 5. The `for`
```

Comment:
It'd be clearer to just say it's a `UnitRange` or maybe even just "a range" (lowercase).